tools/java/runfiles/RunfilesForStardoc.java

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.devtools.build.runfiles;
+
+public class RunfilesForStardoc {
+  public static String getCanonicalRepository(Runfiles runfiles, String sourceRepository) {
```

Comment:
linter wants a javadoc on public methods :)

Reply:
and on the class

Reply:
Added. I also renamed the parameter and added back the `exports` to simplify the diff.